Adaboost/adaboost_sim.py

```diff
@@ -133,8 +152,15 @@ def weakClassifier_error(i,j,k): #Returns the error on the data for a given dist
 
 		
 
-	
+#Now plot the generated labels	
+pos1 = np.nonzero(final_label == 1)
+pos2 = np.where(final_label== -1)
 
+plt.plot(x[pos1,0], x[pos1,1], 'b*')
+plt.plot(x[pos2,0], x[pos2,1], 'r*')
+plt.axis([-3,3,-3,3])
+plt.title("Generated data")
+plt.show()
 
 

```

Comment:
A code will look neat in PEP8 format. Use the tool called autopep8, will solve this